### PR TITLE
Add QR code to new connection page on faber and acme controller

### PR DESCRIPTION
**Pull request by ClemTheStudent**
Related to : https://github.com/hyperledger/aries-acapy-controllers/issues/88#issue-1728720771 and https://github.com/hyperledger/aries-acapy-controllers/issues/89

### CODE REVIEW

File: AliceFaberAcmeDemo/controllers/acme-controller/views/new_connection.hbs [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {{> component_nav moreNavLinks=navLinks}}
 
+<!-- Assurez-vous d'inclure le script qrcode.min.js avant d'essayer de l'utiliser -->
+<script src="/javascripts/qrcode.min.js"></script>
```
Comment (amanji) [inline AliceFaberAcmeDemo/controllers/acme-controller/views/new_connection.hbs:4]: Can we reference this from a CDN?

**Comment by amanji** [comment]
You will need to signoff on your commits for the DCO check to pass. To do this you can rebase your commits and amend them using the `-s` flag. You will then need to force push using `--force-with-lease` for this PR to take effect.

**Comment by amanji** [comment]
Thanks for updating your commits. Please see my comment about including the QR code library. If we can simply reference this rather than including it in the codebase, that would be better.

**Comment by ClemTheStudent** [comment]
> 

I added the script from this CDN : https://www.jsdelivr.com/package/npm/davidshimjs-qrcodejs

**Review by amanji — APPROVED** [review]
Great work! Thank you for the contributions.

**Comment by swcurran** [comment]
Awesome stuff — thanks!

**Pull request merged**